json-guides/circuit-breaker.json

```diff
@@ -185,7 +185,7 @@
           "title": "Configuring a delay",
           "TOCIndent": 1,
           "description": [
-              "By default, the circuit remains open for <code>5000 ms</code>. During this period, all requests to the main service are blocked and fail immediately. At the end of this period, the circuit changes to the half-open state and checks if the main service is healthy. This time period can be configured with the delay parameter.",
+            "By default, the circuit remains open for <code>5000 ms</code>. During this period, all requests to the main service are blocked and fail immediately. At the end of this period, the circuit changes to the half-open state, which will be introduced in the 'Configuring the success threshold' step. This time period can be configured with the delay parameter.",
```

Comment:
Change "At the end of this period, the circuit changes to the half-open state, which will be introduced in the 'Configuring the success threshold' step." to "At the end of this period, the circuit changes to the half-open state. This state is introduced in the 'Configuring the success threshold' step.